src/porepy/numerics/fv/mpfa.py

```diff
@@ -174,17 +170,7 @@ def discretize(self, g, data):
 # ------------------------------------------------------------------------------#
 
 
-def mpfa(
-    g,
-    k,
-    bnd,
-    robin_weight=None,
-    eta=None,
-    inverter=None,
-    apertures=None,
-    max_memory=None,
-    **kwargs
-):
+def mpfa(g, k, bnd, eta=None, inverter=None, apertures=None, max_memory=None, **kwargs):
```

Comment:
I guess this is some sort of automatic formatting? I thought the one parameter per line version for many parameters was the version preferred by black. Am I wrong?

Reply:
This is black formatting. It puts the arguments on one line if it can fit them